.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 test/report
 test/tmp
 bower_components
+.idea/
```

Comment:
There are certain projects I like to keep them in, but I have no problem with removing the change :smile: